### PR TITLE
Bump SwiftLint 0.15.0

### PR DESCRIPTION
**Pull request by tbaranes**
New opt-in rules in addition with the _enabled by default rules_ enabled:
 - `attributes`
 - `closure_end_indentation`
 - `first_where`
 - `number_separator`
 - `operator_usage_whitespace`
 - `prohibited_super_call`

### CODE REVIEW

File: IBAnimatable/PanInteractiveAnimator.swift [hunk 1]
```diff
@@ -60,7 +60,7 @@ public class PanInteractiveAnimator: InteractiveAnimator {
     progress = min(max(progress, 0), 0.99)
 
     // Finish the transition when pass the threathold
-    let shouldFinishInteractiveTransition =  progress > 0.5 || speed > 1000
+    let shouldFinishInteractiveTransition = progress > 0.5 || speed > 1_000
```
Comment (JakeLin) [inline IBAnimatable/PanInteractiveAnimator.swift:63]: I don't quite understand why it is `1_000`?

**Review by JakeLin — APPROVED** [review]
Just a small comment. otherwise, it is great 👍, thanks for updating that.

File: IBAnimatable/PanInteractiveAnimator.swift (hunk 1 above)
Comment (tbaranes) [reply IBAnimatable/PanInteractiveAnimator.swift:63]: That's due to `number_separator`: separate big numbers by "_" to make it easier to read. I'm not a huge fan for 1000, but starting with the next version, we can configure it to warn for number only > 10_000

Comment (JakeLin) [reply IBAnimatable/PanInteractiveAnimator.swift:63]: @tbaranes OK, cool.

**Pull request merged**